cassandra/pom.xml

```diff
@@ -88,8 +88,8 @@
         </dependency>
 
         <dependency>
-            <groupId>net.jpountz.lz4</groupId>
-            <artifactId>lz4</artifactId>
+            <groupId>org.lz4</groupId>
+            <artifactId>lz4-java</artifactId>
```

Comment:
This is required by cassandra java driver 3.7.2 ?

Reply:
@zjffdu stricktly speaking, these are [optional dependencies](https://docs.datastax.com/en/developer/java-driver/3.7/manual/compression/), but because they were used in `pom.xml` already, I just changed them to match versions used in the driver.